scripts/commands/gotonpc.lua

```diff
@@ -0,0 +1,25 @@
+---------------------------------------------------------------------------------------------------
+-- func: gotonpc
+-- auth: Chronos77
+-- desc: Goes to the target player.
+---------------------------------------------------------------------------------------------------
+
+cmdprops =
+{
+    permission = 1,
+    parameters = "s"
+};
+
+function onTrigger(player, target)
+    if (target == nil) then
+        player:PrintToPlayer("You must enter a valid npc id.");

```

Comment:
Yeah its pretty impractical (near impossible from all the caveats involved) to do it by name.